test_weather_data_fetcher.py

```diff
@@ -0,0 +1,61 @@
+import os
+import dotenv
+from unittest import mock
+
+from weather_data_fetcher import fetch_location_coords, fetch_weather_data, get_api_key
+
+
+class TestWeatherDataFetcher:
+    def test_fetch_coords(self):
+        coords_berlin = fetch_location_coords("Berlin")
+        coords_difference = [sum(x) for x in zip(coords_berlin, (-52.520008, -13.404954))]
+        assert sum(list(coords_difference)) < 0.01
```

Comment:
Interesting way to test this :sweat_smile: is there maybe an easier and more transparent way to possible test this? Like checking if something equals something without summing up and other tricks?